backend/testUtils/testSession.ts

```diff
@@ -1,10 +1,49 @@
-const mockTestSession = {
+import {
+  ResultRequestDTO,
+  TestSessionRequestDTO,
+  TestSessionResponseDTO,
+} from "../services/interfaces/testSessionService";
+
+export const testResult: ResultRequestDTO = {
+  student: "some-student-name",
+  score: 25,
+  answers: [10, 11],
+  breakdown: [false, true, true],
+};
+
+export const mockTestSession: TestSessionRequestDTO = {
   test: "62c248c0f79d6c3c9ebbea95",
   teacher: "62c248c0f79d6c3c9ebbea94",
   school: "62c248c0f79d6c3c9ebbea93",
   gradeLevel: 4,
+  results: [testResult],
   accessCode: "1234",
   startTime: new Date("2021-09-01T09:00:00.000Z"),
 };
 
-export default mockTestSession;
+export const assertResponseMatchesExpected = (
+  expected: TestSessionRequestDTO,
+  result: TestSessionResponseDTO,
+): void => {
+  expect(result.id).not.toBeNull();
+  expect(result.test.toString()).toEqual(expected.test);
+  expect(result.teacher.toString()).toEqual(expected.teacher);
+  expect(result.school.toString()).toEqual(expected.school);
```

Comment:
did we decide on whether we want these to return the actual objects rather than just the IDs? if so, we can create a new ticket for that and do it in a separate PR

Reply:
i believe we are returning the actual objects - i was thinking we could use `getTestById`, `getTeacherById` and `getSchoolById` once they are implemented

Reply:
@carissa-tang sorry if I'm misunderstanding, but if we end up using those functions, shouldn't we just return Ids instead of objects?

Reply:
@joyce-shi within `createTestSession` we need to validate the 3 given ids and so we were thinking if we are already making the call, we can map the objects to the ids in the respone. is there a downside you are considering with this approach?

Reply:
oh shoot, i forgot about this. makes sense! sounds good.